app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigStageConfig.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { SETTINGS } from 'core/config/settings';
+import { IStageConfigProps, FormikStageConfig } from 'core/pipeline';
+import { FormikFormField, TextInput } from 'core/presentation';
+import { HelpField } from 'core/help';
+
+export const ImportDeliveryConfigStageConfig: React.SFC<IStageConfigProps> = stageConfigProps => (
```

Comment:
I _think_ this should work fine, and makes the type signature slightly more resilient to changes.

```suggestion
export const ImportDeliveryConfigStageConfig: React.ComponentType<IStageConfigProps> = stageConfigProps => (
```